src/main/java/FileReaderWriter.java

```diff
@@ -2,6 +2,7 @@
 import java.nio.charset.StandardCharsets;
 import java.nio.file.Files;
 import java.util.Objects;
+import java.util.Scanner;
```

Comment:
I noticed that you've switched from using `BufferedReader` to `Scanner`, which makes the code more concise and the file reading logic clearer. I like this improvement! Did you choose `Scanner` because of specific features that better fit the current needs, or was it mainly to reduce the complexity of the code?